geopandas/plotting.py

```diff
@@ -587,6 +587,27 @@ def plot_dataframe(
     -------
     ax : matplotlib axes instance
 
+    Examples
+    --------
+    >>> df = geopandas.read_file(geopandas.datasets.get_path("naturalearth_lowres"))
+    >>> df.head()  # doctest: +SKIP
+        pop_est      continent                      name iso_a3  \
+gdp_md_est                                           geometry
```

Comment:
Maybe keep it on a single line? (the output you would get in a wider console)

Reply:
This is the output you would get in a wider console, that is why it is skipped in fact. It is a single line in the docs, I just had to break it here due to linting limit of 88 characters.